### PR TITLE
added algo to count inversions in an array in C++, Python, JavaScript and Java

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: arrays/python/count-inversions.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  mid = int(size/2)
+  left = arr[:mid]
+  right = arr[mid:]
+  l_inv = count_inversions(left)
+  r_inv = count_inversions(right)
+  split_inv = count_split_inv(arr, left, right)
+
+  return l_inv + r_inv + split_inv
+
+with open("inp.txt") as f:
```
Comment (Arsenic-ATG) [inline arrays/python/count-inversions.py:41]: preferred place to take user input should be stdin instead of a separate input file.

**Comment by atin** [comment]
@Arsenic-ATG @paulsonjpaul Updated the files. Should be okay now.

**Review by Arsenic-ATG — COMMENTED** [review]
following files works and are implemented correctly. 
- [x] README.md 
- [x] count-inversions.cpp
- [x] count-inversions.py 
- [ ] count-inversion.java
- [ ] count-inversion.js

(`js` and `java` versions works functionally, implementation also looks ok but let @paulsonjpaul and @ming-tsai also review them as they are more familiar with java and js than me )

**Comment by atin** [comment]
@ming-tsai @paulsonjpaul Have you guys reviewed the files? Just reminding!😅😀

**Review by ming-tsai — APPROVED** [review]
Nice work @atin

**Pull request merged**